packages/media/src/generateVideoStrip.js

```diff
@@ -58,6 +58,7 @@ async function generateVideoStrip(element, resource, stripWidth, stripHeight) {
     scale,
     focalX,
     focalY,
+    flip,
```

Comment:
Hm, there's one issue currently with the video strip -- it looks like if flipping while in trim mode, it doesn't update the strip, only the image on the Page gets updated.

Reply:
Got it. Should be fixed now although takes a while to regenerate the strip in the correct orientation if you change the flip settings while in flip mode

Reply:
Hm, I wonder if there's a way to reuse the previous images (store the originals after generation) and flip as needed before putting together?
I assume the frame generation is the main thing that takes time? Since we should need the same frames (but just flipped) then could we reuse these. WDYT?

Reply:
Yes, correct generating the frames takes a bit of time but am thinking there are way too many variables to make it useful to store the generated images. position, size , rotation, length and flip settings of both the video and canvas  all factor in the generated frames.

Reply:
Isn't it possible to store the frames as they were originally before the changes in this PR, and only flip it afterwards for strip generation if needed?

Reply:
(We can live with it taking a bit of time to flip the strip if it gets very complicated, asking just in case though!)

Reply:
I do think it would be possible but not sure if the benefits outweigh the added complexity since we would need to regenerate the frames anyway incase any of the other variables changed  but lets ping @barklund to see if he has more clarity on this since he knows the feature a bit more incase am wrong

Reply:
I'm not sure I understand the question here. What is the exact goal and what are the possible solutions and their pros and cons?

I need this spelled out in big cardboard letters here to follow along 🙈 .

Reply:
@barklund we are just discussing with Miiina if it would be good to save and reuse the generated frames for the video strip. So we would save the frames the first time we generate them and then instead of re-generating them on subsequent renders, we would use the stored frames. The  pros would be the frames strip would render faster since we would eliminate the need to generate the frames all the time but the catch is that the frames generated depend on so many variables like the position of the video in the canvas, the size, orientation , rotation such tham am thinking we would mostly need to generate the video most of the time anyway due to some of these variables changing. Too much rumbling, hope am making sense 😄

Reply:
@barklund While a user is in trim mode and changes something of the video (e.g. flips the video), the strip is fully regenerated and updating the strip takes some time. However, the origin video is actually the same, and the generated frames as well, so the idea was that perhaps we could store the initial frames and then reuse those if possible instead.

(If doing that gets too complicated, we can just leave it as-is, too, it's already working well as it is)

Reply:
Hmm, as Jasper mentions, there are too many things that would go into the cache as parameters, so I don't think it's worth it.

Reply:
Alright, sgtm! @Trapsta is the PR otherwise ready for review (since it was moved back to In Progress yesterday)?

Reply:
@miina yes,  will update the status